AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts

```diff
@@ -401,11 +401,6 @@ describe('AddParticipantComponent', () => {
         lastName.setValue('María Jose Carreño Quiñones');
         expect(lastName.valid).toBeFalsy();
     });
-    it('should validate phone', () => {
```

Comment:
do we not want to validate the input when it is not an empty string?

Reply:
Yeah you're right, ill update it